src/Java.Interop/Java.Interop/JniRuntime.JniValueManager.cs

```diff
@@ -70,7 +71,7 @@ protected virtual void Dispose (bool disposing)
 
 			public abstract void Finalize (IJavaPeerable value);
 
-			public abstract List<JniSurfacedPeerInfo>   GetSurfacedPeers ();
+			public abstract Collection<JniSurfacedPeerInfo>   GetSurfacedPeers ();
```

Comment:
Why change to `Collection<JniSurfacedPeerInfo>`?

Reply:
Ah, because of:

```
Gendarme.Rules.Design.Generic.DoNotExposeGenericListsRule
```

That rule is stupid. ;-)

Reply:
That's right, we don't hold the List, so we don't care what consumer code does with it.